contracts/RewardManager.sol

```diff
@@ -219,15 +249,6 @@ contract RewardManager is Ownable, Versionable, Safe {
     return owners[0] == address(this) ? owners[1] : owners[0];
   }
 
-  function getTransferRewardSafeData(
```

Comment:
I inlined this function and saved some bytes, also some execution gas I think:



Before inline  getWithdrawFromRewardSafeData:
```
╰─➤ yarn test:size | grep RewardManager
 |  contracts/RewardManager.sol:RewardManager                                                            ·     14.623  │
```
After inline  getWithdrawFromRewardSafeData:
```
╰─➤ yarn test:size | grep RewardManager
 |  contracts/RewardManager.sol:RewardManager                                                            ·     14.598  │
```
After also inline getTransferRewardSafeData:
```
╰─➤ yarn test:size | grep RewardManager
 |  contracts/RewardManager.sol:RewardManager                                                            ·     14.505  │
```

Reply:
kewl!